lib/primer/classify.rb

```diff
@@ -94,19 +94,18 @@ class Classify
 
     class << self
       def call(classes: "", style: nil, **args)
-        extracted_results = extract_hash(args)
-
-        extracted_results[:class] = [
-          validated_class_names(classes),
-          extracted_results.delete(:classes)
-        ].compact.join(" ").strip.presence
-
-        extracted_results[:style] = [
-          extracted_results.delete(:styles),
-          style
-        ].compact.join.presence
-
-        extracted_results
+        extract_css_attrs(args).tap do |extracted_results|
+          classes = +"#{validated_class_names(classes)} #{extracted_results[:class]}"
+          classes.strip!
+          # rubocop:disable Rails/Blank
+          # do this instead of using presence/blank?, which are a lot slower
+          extracted_results[:class] = !classes || classes.empty? ? nil : classes
```

Comment:
for the sake of readability, I wonder if we could create our "presence" method instead of repeating the logic

```rb
def presence(obj)
  !obj || obj.empty? ? nil : classes
end
```

Reply:
Ordinarily I'd agree with you, but I'd like to avoid any additional method call overhead. Lemme put it in tho and see how the numbers look.

Reply:
Looks like the additional method adds a negligible amount of overhead 🎉